### PR TITLE
Update convert-source-map to 1.1.1 to fix stack overflow issues

### PR DESCRIPTION
**Pull request by theogravity**
Unit tests pass:

```
[theo@MacBook-Pro node-enchilada (master)]$ npm run test

> enchilada@0.12.0 test /Volumes/sixfive-cs/node-enchilada
> mocha test/*.js

(node) child_process: options.customFds option is deprecated. Use options.stdio instead.

  ․ basic setup: 5ms
  ․ basic basic: 91ms
  ․ compress setup: 0ms
  ․ compress /foo.js - not entry: 109ms
  ․ routes setup: 1ms
  ․ routes /foo.js - should run: 39ms
  ․ routes /baz.js - entry: 52ms
  ․ routes /cats-module.js - should run: 40ms
  ․ routes /cats.js - entry: 45ms
  ․ server setup normal http server: 0ms
  ․ server basic without express: 36ms
  ․ server handle browserify errors: 15ms
  ․ server handle missing file: 2ms
  ․ sourcemap setup: 1ms
  ․ sourcemap basic: 48ms
  ․ sourcemap setup for compress: 1ms
  ․ sourcemap basic: 89ms
  ․ transforms setup: 0ms
  ․ transforms transform called: 34ms

  19 passing (624ms)

```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "enchilada",
-  "version": "0.12.0",
+  "version": "0.12.1",
```
Comment (defunctzombie) [inline package.json]: please don't do this

File: package.json [hunk 2]
```diff
@@ -21,12 +21,12 @@
   "author": "Roman Shtylman <shtylman@gmail.com>",
   "license": "MIT",
   "dependencies": {
-    "mime": "1.2.11",
     "browserify": "4.1.9",
-    "filewatcher": "1.1.1",
-    "uglify-js": "2.4.13",
+    "convert-source-map": "1.1.1",
```
Comment (defunctzombie) [inline package.json]: any reason for the re-ordering here? Makes the diff more annoying to review :)

Comment (marcello3d) [inline package.json]: it's probably from `npm --save`. it sorts dependencies alphabetically so that you don't run into the annoying diff problem. :-)

**Comment by theogravity** [comment]
Rebased to remove the version change. The reordering is indeed from doing `npm --save`. 

Not intending to be rude, but how is it easy to read the package list in an unordered state?

**Comment by defunctzombie** [comment]
LGTM

**Comment by defunctzombie** [comment]
@theogravity I just use my editor search features so don't really think about the alphanumeric ordering in the file.

**Comment by theogravity** [comment]
Ah, gotcha. Thanks!

**Pull request merged**